libs/api/domains/auth/src/lib/dto/updateConsent.input.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType('UpdateAuthConsentInput')
+export class UpdateConsentInput {
```

Comment:
As the mutation is named `patchAuthConsent` we should probably reflect that in the input name.
```suggestion
@InputType('PatchAuthConsentInput')
export class PatchConsentInput {
```

Reply:
Fixed.